HandleService/app/src/main/java/br/com/handleservice/presentation/screens/profile/ProfileScreen.kt

```diff
@@ -1,22 +1,152 @@
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.*
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import br.com.handleservice.R
 
 @Composable
 fun ProfileScreen(
-    modifier: Modifier = Modifier,
+    viewModel: ProfileViewModel = hiltViewModel()
 ) {
+    val profileState = viewModel.profileState.collectAsState().value
+
     Column(
-        modifier = modifier.fillMaxSize(),
-        verticalArrangement = Arrangement.Center,
-        horizontalAlignment = Alignment.CenterHorizontally
+        modifier = Modifier
+            .fillMaxSize()
+            .background(colorResource(R.color.background))
+            .padding(horizontal = 16.dp),
+    ) {
+        Spacer(modifier = Modifier.height(24.dp))
+
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 16.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Image(
+                painter = painterResource(R.drawable.profile_image_fallback),
+                contentDescription = "Foto de Perfil",
+                modifier = Modifier
+                    .size(70.dp)
+                    .clip(CircleShape),
+                contentScale = ContentScale.Crop
+            )
+            Spacer(modifier = Modifier.width(16.dp))
+            Text(
+                text = profileState.name,
+                style = MaterialTheme.typography.titleLarge.copy(fontWeight = FontWeight.Bold),
+                fontSize = 20.sp,
+                color = colorResource(R.color.handle_titles)
+            )
+        }
+
+        Spacer(modifier = Modifier.height(12.dp))
+
+        ProfileOption(
+            icon = R.drawable.ic_filled_messages,
+            label = "Chat",
+            description = "Meu histórico de conversas",
+            onClick = { }
+        )
+        ProfileOption(
+            icon = R.drawable.ic_filled_address,
+            label = "Meus Endereços",
+            description = "Endereços cadastrados",
+            onClick = { }
+        )
+        ProfileOption(
+            icon = R.drawable.ic_filled_document_data,
+            label = "Dados da Conta",
+            description = "Minhas informações",
+            onClick = { }
+        )
+        ProfileOption(
+            icon = R.drawable.ic_settings,
+            label = "Configurações",
```

Comment:
Sempre que for importar um ícone, seleciona a opção rounded no Material Symbols. A Mari está usando isso em praticamente todos os ícones:

<img width="401" alt="image" src="https://github.com/user-attachments/assets/d90fed66-1508-4bac-b9ae-a5649bcf304e">